LICENSE.md

```diff
@@ -1,25 +1,28 @@
-The Lingua-Franca toolkit is licensed under the BSD 2-Clause License:
+This software is licensed under the BSD 2-Clause License.
 
-Copyright (c) 2019-2020, Industrial Cyberphysical Systems (iCyPhy). All rights reserved.
+Copyright (c) 2019-present, The Lingua Franca contributors. All rights reserved.
```

Comment:
```suggestion
Copyright © 2019-2022, The Lingua Franca contributors. All rights reserved.
```

Reply:
@edwardalee do you think the capitalization here checks out?

Reply:
Looks good to me.  Is the non-ASCII character going to bite (byte?) us?